src/cbapi/psc/livequery/rest_api.py

```diff
@@ -32,3 +32,6 @@ def _perform_query(self, cls, **kwargs):
 
     def query(self, sql):
         return self.select(Run).where(sql=sql)
+
+    def query_runs(self, query=None):
```

Comment:
Instead of `query_runs` maybe `query_history` as they are both are for runs

Reply:
Changing it.